src/modules/tensor/cpu/kernel/blend.cpp

```diff
@@ -339,6 +339,10 @@ RppStatus blend_f32_f32_host_tensor(Rpp32f *srcPtr1,
                     p1[0] = _mm_fmadd_ps(_mm_sub_ps(p1[0], p2[0]), pMul, p2[0]);    // alpha-blending adjustment
                     p1[1] = _mm_fmadd_ps(_mm_sub_ps(p1[1], p2[1]), pMul, p2[1]);    // alpha-blending adjustment
                     p1[2] = _mm_fmadd_ps(_mm_sub_ps(p1[2], p2[2]), pMul, p2[2]);    // alpha-blending adjustment
+                    //boundary checks for f32
+                    p1[0] = rpp_pixel_check_0to1_sse(p1[0]);
+                    p1[1] = rpp_pixel_check_0to1_sse(p1[1]);
+                    p1[2] = rpp_pixel_check_0to1_sse(p1[2]);
```

Comment:
Isn't there a better way to mask this explicit pixel check instead of checking these 3 lines for all instances? Are we doing this for U8 too?